GsaGH/Components/5_Results/Elem1dResultDiagram.cs

```diff
@@ -255,17 +223,17 @@ string warningText
           $"Model came straight out of GSA and we couldn't read the units. The geometry has been scaled to be in {lengthUnit}. This can be changed by right-clicking the component -> 'Select Units'");
       }
 
-      double computedScale = ComputeScale(result.Model, scale);
+      double computedScale = ComputeScale(result, scale, _lengthUnit, autoScale);
```

Comment:
we don't have to add lengthUnit arg here, we have access to this field everywhere in this class

Reply:
`ComputeScale` method has been moved out of this class to be called from any future class using GsaAPI Graphics / Diagram

Reply:
but here you are still using the method declared in elem1dResultDiagram
![image](https://github.com/arup-group/GSA-Grasshopper/assets/83698580/df34e67a-7f01-4b7c-bb40-31829decbab6)

Reply:
Yes it's not refactored yet - but should eventually be. It's entangled with IsForce, IsMoment, IsStress, so we need an input for that as well if we separate this function out. What do you suggest?

Reply:
isForce method etc. would leave it here. 
computeScale method would add to some interface. because implementations of this method will be different for elem1d, elem2d etc.